src/Domain/User/ValueObject/Email.php

```diff
@@ -4,34 +4,22 @@
 
 namespace App\Domain\User\ValueObject;
 
+use App\Domain\Shared\ValueObject\AbstractString;
 use Assert\Assertion;
```

Comment:
This import isn't needed anymore

Reply:
This is required for `create` method, where the assertion is still performed

Reply:
Oh right! sorry